/cf_1/aruco_node:
  ros__parameters:
    marker_size: 0.1
    aruco_dictionary_id: DICT_5X5_250
    image_topic: /cf_1/image
    camera_info_topic: /cf_1/camera_info
    camera_frame: world

/cf_2/aruco_node:
  ros__parameters:
    marker_size: 0.1
    aruco_dictionary_id: DICT_5X5_250
    image_topic: /cf_2/image
    camera_info_topic: /cf_2/camera_info
    camera_frame: world

/cf_3/aruco_node:
  ros__parameters:
    marker_size: 0.1
    aruco_dictionary_id: DICT_5X5_250
    image_topic: /cf_3/image
    camera_info_topic: /cf_3/camera_info
    camera_frame: world

/cf_4/aruco_node:
  ros__parameters:
    marker_size: 0.1
    aruco_dictionary_id: DICT_5X5_250
    image_topic: /cf_4/image
    camera_info_topic: /cf_4/camera_info
    camera_frame: world

